doc/http.qbk

```diff
@@ -41,76 +41,236 @@ both Boost.Asio and the HTTP protocol specification described in
 
 
 
-[section:example Example]
+[section:usage Usage]
+
+[note
+    Sample code and identifiers mentioned in this section are written
+    as if the following declarations are in effect:
+    ```
+    #include <beast/http.hpp>
+    using namespace beast;
+    ```
+]
+
+A HTTP message (referred to hereafter as "message") contains a request or
+response line, a series of zero or more name/value pairs (collectively
+termed "headers"), and a series of octets called the message body which may
+be zero in length. The HTTP protocol defines the client and server roles:
+clients send messages called requests and servers send back messages called
+responses. `http::message` models both requests and responses. The library
+provides interfaces to perform these operations on messages:
+
+* [*Parse] a new message from a series of octets.
+
+* [*Assemble] a new message from scratch or from an existing message.
+
+* [*Serialize] a message into a series of octets.
+
+* [*Read] a message from a stream. This can be thought of as a compound
+operation; a network read, followed by a [*parse].
+
+* [*Write] a message to a stream. This can be thought of as a compound
+operation: a [*serialize] followed by a network write.
 
-All examples and identifiers mentioned in this document are written as
-if the following declarations are in effect:
+In the paragraphs that follow we describe simple interfaces that will serve
+the majority of users looking merely to interact with a HTTP server, or
```

Comment:
It looks like you are probably using "a HTTP" consistently.  If that's true you might as well leave it that way.